ar.json

```diff
@@ -168,7 +168,7 @@
   "data.chainstate": "حالة السلسلة",
   "data.chainstate.updated_daily": "مُحدَّث يوميًّا",
   "data.chainstate.addresses": "العناوين",
-  "data.chainstate.utxo_count": "عدد UTXO",
+  "data.chainstate.utxo_count": "عدد المخرجات غير المنفقة",
```

Comment:
This is supposed to be a short label... It previously just said "UTXOs", now its "UTXO Count". I'm afraid this change has way too many characters and will break the layout in that Chainstate section in the sidebar. Can you shorten this?